datalake/src/test/java/com/sequenceiq/datalake/service/sdx/SdxUpgradeServiceTest.java

```diff
@@ -31,6 +32,12 @@
 @ExtendWith(MockitoExtension.class)
 public class SdxUpgradeServiceTest {
 
+    private static final String USER_CRN = "crn:cdp:iam:us-west-1:1234:user:1";
+
+    private static final String CLUSTER_NAME = "coolStack";
+
+    private static final String OS_UPGRADE_CHECK_EXCEPTION_MESSAGE = "OS upgrade can't be done because the cluster is not in an available state!";
```

Comment:
Are these being used anywhere?

Reply:
Nope. However, since someone is waiting for the merge and since this is a small change only, I would not correct it, but add a jira for that, if that is ok.